neat-form-core/src/main/java/com/nerdstone/neatformcore/internationalization/LanguageHelper.kt

```diff
@@ -0,0 +1,36 @@
+package com.nerdstone.neatformcore.internationalization
+
+import org.apache.commons.text.StringSubstitutor
+import java.util.*
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.jvm.isAccessible
+
+/**
+ * Created by ndegwamartin on 25/04/2021.
+ *
+ * A class containing helper methods for internationalization and localization support
+ */
+object LanguageHelper {
+
+    fun getBundleStringSubstitutor(baseName: String, locale: Locale?): StringSubstitutor? {
```

Comment:
Will the `locale` param be `null` at any point? If we are never expecting a `null` argument then I think it is safe to make it not null.

Reply:
Agreed